FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v4.3.2
+- [fixed] Fixed broken callback to AppDelegate after retrieveing a dynamic link during fresh app start.
```

Comment:
Changelog is unneeded since this bug hasn't yet released.

Reply:
Done